apps/chat-e2e/src/tests/chatBarFolderConversation.test.ts

```diff
@@ -685,3 +686,256 @@ dialTest(
+      async () => {
+        await chatBar.dragAndDropEntityToFolder(
+          folderConversations.getFolderByName(
+            ExpectedConstants.newFolderWithIndexTitle(2),
+            2,
+          ),
+          conversations.chronologyByTitle(Chronology.today),
+        );
+
+        await chatBarFolderAssertion.assertRootFolderState(
+          { name: ExpectedConstants.newFolderWithIndexTitle(2) },
+          'visible',
+        );
+      },
+    );
+
+    await dialTest.step(
+      'Context menu appears if to click on chat name or folder name using right mouse button',
+      async () => {
+        // Chat context menu
+        await conversations
+          .getEntityByName(firstConversation.name)
+          .click({ button: 'right' });
+        await conversationDropdownMenuAssertion.assertMenuState('visible');
+        await conversations.getConversationName(firstConversation.name).click();
+
+        // Folder context menu
+        await folderConversations
+          .getFolderByName(ExpectedConstants.newFolderWithIndexTitle(1))
+          .click({ button: 'right' });
+        await folderConversations
+          .getDropdownMenu()
+          .waitForState({ state: 'visible' });
+      },
+    );
+  },
+);
+
+dialTest(
+  'Menu for conversation in Replay mode\n' +
+    'Single chat can not be stored in Pinned chats section',
+  async ({
+    dialHomePage,
+    conversationData,
+    dataInjector,
+    conversations,
+    conversationDropdownMenu,
+    setTestIds,
+    localStorageManager,
+    conversationAssertion,
+    chatBar,
+    chatBarFolderAssertion,
+    conversationDropdownMenuAssertion,
+  }) => {
+    setTestIds('EPMRTC-1138', 'EPMRTC-1598');
+    const conversation = conversationData.prepareDefaultConversation();
+    conversationData.resetData();
+    const replayConversation =
+      conversationData.prepareDefaultReplayConversation(conversation);
+    await dataInjector.createConversations([conversation, replayConversation]);
+    await localStorageManager.setSelectedConversation(conversation);
+
+    await dialTest.step(
+      'Create New conversation and send any message there',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded();
+      },
+    );
+
+    await dialTest.step('Open menu for the conversation', async () => {
+      await conversations.openEntityDropdownMenu(
+        ExpectedConstants.replayConversation + conversation.name,
+      );
+      const expectedMenuOptions = [
+        MenuOptions.select,
+        MenuOptions.rename,
+        MenuOptions.duplicate,
+        MenuOptions.export,
+        MenuOptions.moveTo,
+        MenuOptions.delete,
+      ].map((option) => option.toString());
+      await conversationDropdownMenuAssertion.assertMenuIncludesOptions(
+        ...expectedMenuOptions,
+      );
+
+      const actualMenuOptions =
+        await conversationDropdownMenu.getAllMenuOptions();
+      expect
```

Comment:
please, remove as duplicated assertion - the same is done above

Reply:
done